components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -500,6 +502,17 @@
                                     </td>
                                 </tr>
                                 <% } %>
+                                <tr id="bypass_client_credentials">
+                                    <td colspan="2">
+                                        <label>
+                                            <input type="checkbox" name="bypass_client_credentials" value="yes">
```

Comment:
Need to close the tag, 'input'.

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1058